src/js/utils/bounds.js

```diff
@@ -143,10 +143,11 @@ export function getVisibleBounds(body, target) {
 }
 
 export function removeUtilElements(body) {
-  if (zeroElement) {
-    body.removeChild(zeroElement);
-  }
-  zeroElement = null;
+    if (zeroElements.has(body)) {
+        body.removeChild(zeroElements.get(body));
```

Comment:
Looks like we have some spacing issues here.

Reply:
Should be fixed now